indigo/release-armhf-build.yaml

```diff
@@ -2,41 +2,16 @@
 # ROS buildfarm release-build file
 ---
 abi_incompatibility_assumed: true
-jenkins_binary_job_priority: 82
+jenkins_binary_job_priority: 83
 jenkins_binary_job_timeout: 120
```

Comment:
I updated the timeout for now but if the ARMhf jobs will actually require that many resources I would reconsider building them on our farm due to resources.